src/frontend/citizen-portal/src/app/services/dispute.service.ts

```diff
@@ -32,3 +32,10 @@ export class DisputeService {
     })
   }
 }
+
+export enum StatusStepType {
+  SUBMITTED = "Submitted",
+  PROCESSING = "Processing",
+  REVIEW = "Review/Scheduling",
```

Comment:
Instead of REVIEW the third step should read HEARING = "Hearing Scheduled" as per the story

Reply:
updated